src/norm/private/postgres/rowutils.nim

```diff
@@ -21,20 +21,20 @@ func isEmptyColumn*(row: Row, index: int): bool =
 
 ## This does the actual heavy lifting for parsing
 proc fromRowPos[T: ref object](obj: var T, row: Row, pos: var Natural, skip: static bool = false) =
-  ##[ Convert ``ndb.sqlite.Row`` instance into `ref object`_ instance, from a given position.
```

Comment:
I'd disagree with this doc comment change, as it is no longer solely accurate. With the move to ref object in order to allow rawSelect, this does more than just convert Models